Tests/Data/ThermoMechanics/InitialStates/into_initial_state.prj

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <meshes>
+        <mesh>soil_column_q8.vtu</mesh>
+        <mesh>soil_column_bottom.vtu</mesh>
+        <mesh>soil_column_top.vtu</mesh>
+        <mesh>soil_column_left.vtu</mesh>
+        <mesh>soil_column_right.vtu</mesh>
+    </meshes>
+    <processes>
+        <process>
+            <name>TM</name>
+            <type>THERMO_MECHANICS</type>
+            <integration_order>3</integration_order>
+            <constitutive_relation>
+                <type>LinearElasticIsotropic</type>
+                <youngs_modulus>E</youngs_modulus>
+                <poissons_ratio>nu</poissons_ratio>
+            </constitutive_relation>
+            <reference_solid_density>rho_sr</reference_solid_density>
+            <linear_thermal_expansion_coefficient>alpha</linear_thermal_expansion_coefficient>
+            <specific_heat_capacity>cs</specific_heat_capacity>
+            <thermal_conductivity>lambda</thermal_conductivity>
+            <specific_body_force>0 -9.81</specific_body_force>
```

Comment:
Merged but these values have to be set as 0.0 0.0

Reply:
I don't think so. It's only in the non-equilibrium restart, where the stresses due to gravity are included in the initial non-equilibrium stress. But in the forward tests the gravity is present.
Compare with the SD tests: https://github.com/ufz/ogs/blob/master/Tests/Data/Mechanics/InitialStates/into_initial_state.prj#L21

Reply:
I see.